pages/content/amp-dev/documentation/components/index.html

```diff
@@ -78,22 +106,86 @@ <h1>The AMP component catalogue</h1>
         </div>
       </div>
 
-      <p>If you want to know more about how AMP handles those under the hood read on with <a href="{{ g.doc('/content/amp-dev/about/how-amp-works.html', locale=doc.locale).url.path }}">How AMP works</a>.</p>
-
       <hr>
 
-      {% set categories = g.categories('/content/amp-dev/documentation/components/reference', locale=doc.locale) %}
-      {% include '/views/partials/category-filter.j2' %}
+      {% do doc.styles.addCssFile('css/components/molecules/teaser-component-code.css') %}
+      <section class="ap--section">
+        <h2>Highlighted Components</h2>
+        <div class="ap-m-teaser-component">
+          <div class="ap-m-teaser-container">
+            {% for teaser in doc.highlight_teaser %}
+            <div class="ap-m-teaser ap-m-teaser-code">
+              <a href="{{ teaser.url }}">
+                <div class="ap-m-teaser-card">
+                  <div class="ap-m-teaser-tags">
+                    {% for tag in teaser.tags %}
+                    <div class="ap-m-tag ap-m-tag-{{ tag }}"></div>
+                    {% endfor %}
+                  </div>
+                  <div class="ap-m-teaser-content">
+                    <div class="ap-m-teaser-content-text">
+                      <code><span>{{ teaser.title }}</span></code>
+                      <p>{{ teaser.description }}</p>
+                    </div>
+                  </div>
+                </div>
+              </a>
+            </div>
+            {% endfor %}
+          </div>
+        </div>
+      </section>
+
+
+      <section class="ap-m-table-component-container">
+        {% set categories = g.categories('/content/amp-dev/documentation/components/reference', locale=doc.locale) %}
+        {% do doc.styles.addCssFile('/css/components/molecules/table-component.css') %}
+
+
+        <h2>Categories</h2>
+        {% include 'views/partials/category-filter.j2'%}
+
 
-      {% do doc.styles.addCssFile('css/components/organisms/teaser-grid.css') %}
-      <div class="ap-o-teaser-grid">
-        {# Hardcode to a length of 9 teasers as there will be at least that much #}
-        <div class="ap-o-teaser-grid-list ap-o-teaser-grid-list-count-9">
-          {% for teaser_doc in g.collection('/content/amp-dev/documentation/components/reference').list_docs(locale=doc.locale, order_by='title') %}
-            {% include '/views/partials/teaser.j2' %}
-          {% endfor %}
+        {% for category, docs in categories %}
+        {% set scope = namespace(collectedFormats = []) %}
```

Comment:
`collectedFormats` makes sense programatically, though I'd still vote for `categoryFormats` as this is what the variable really holds 🙂